spec/lib/farscape/agent_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+
+describe Farscape::Agent do
+  describe '#instance' do
+
```

Comment:
[nits] rubocop is going to dislike those empty lines but I guess the rest of the lib is far from being void of such things.